### DISCUSSION TITLE
How to add border around view + arrow tool tip

### DISCUSSION THREAD

**Discussion by rankinit** [skydoves/Balloon#167] [Q&A]
I see how to change background color by using `setBackgroundColorResource` how can I add a offsetting border (color) around the view (including the arrow)?

I tried using a custom drawable but this does not work with the arrow.

Thanks!

**Comment by skydoves** [comment]
Hi, actually [this issue](https://github.com/skydoves/Balloon/issues/38) has been created the last year.
But for implementing this, the whole structure of the layout should be re-implemented or need some tricks. 
I will think about this issue more :)

**Reply by VamsiMadhavH** [reply]
Any updates about this issue?

**Reply by codercoderme** [reply]
any update on this?
